geopandas/sindex.py

```diff
@@ -556,6 +575,30 @@ def query_bulk(self, geometry, predicate=None, sort=False):
 
             return res
 
+        @doc(BaseSpatialIndex.nearest)
+        def nearest(self, geometry):
+            # note - could use _as_geometry_array helper here after #1865
```

Comment:
As mentioned below, check `PYGEOS_GE_010` and if `False` return `NotImplementedError` here.